tests/model/test_cycle_data.py

```diff
@@ -412,6 +423,79 @@ def test_irrelevant_repos_data_removal(
     ).exists()
 
 
+def test_signature_remover(test_perf_signature, test_perf_signature_2, test_perf_data, monkeypatch):
+    monkeypatch.setattr(taskcluster, 'Notify', MagicMock())
```

Comment:
I noticed we haven't patched the Taskcluster model  in `test_irrelevant_repos_data_removal()`, even though the runs the cycling command. Why is that?